exporter.go

```diff
@@ -913,7 +940,7 @@ func (e *Exporter) extractCheckKeyMetrics(ch chan<- prometheus.Metric, c redis.C
 	log.Debugf("allKeys: %#v", allKeys)
 	for _, k := range allKeys {
 		if _, err := doRedisCmd(c, "SELECT", k.db); err != nil {
-			log.Debugf("Couldn't select database %#v when getting key info.", k.db)
+			log.Errorf("Couldn't select database %#v when getting key info.", k.db)
```

Comment:
I think we should let users know about cases where their patterns won't be scaped because of the wrong database.

Reply:
Yeah, that makes sense.